### DISCUSSION TITLE
Is there a way to apply overlay and then save the image?

### DISCUSSION THREAD

**Discussion by cosminbvb** [Project-MONAI/tutorials#725] [Q&A]
Hi,

I was wondering if there is a way to apply the predicted mask on top of the CT scan and then save it with the nifti saver.

Thank you!

**Comment by Nic-Ma** [comment]
Hi @cosminbvb ,

Thanks for your interest here.
I think maybe you can just concat the image and prediction at the channel dim and then call `SaveImage` transform to save it?

Thanks.
